### PR TITLE
Set 3.0/4.2 versions

### PR DESCRIPTION
**Pull request by jlsherrill**
Cherry-pick into:

* [ ] Foreman 2.5 (Satellite 6.10)
* [ ] Foreman 2.4
* [ ] Foreman 2.3 (Satellite 6.9)
* [ ] Foreman 2.1 (Satellite 6.8)

### CODE REVIEW

File: guides/common/attributes.adoc [hunk 1]
```diff
@@ -3,9 +3,9 @@
 :DocState: nightly
 
 // Version numbers
-:ProjectVersion: nightly
+:ProjectVersion: 3.0
 :ProjectVersionPrevious: 2.5
-:KatelloVersion: nightly
+:KatelloVersion: 4.2
 :TargetVersion: 2.4
```
Comment (ekohl) [inline guides/common/attributes.adoc:9]: This looks horribly out of date. Why do we have both a TargetVersion and a ProjectVersion? Then again, it's not like foreman-maintain actually works in upstream, right?

Comment (jlsherrill) [reply guides/common/attributes.adoc:9]: right, i don't think those versions are relevant for upstream docs, and probably aren't relevant at all for a releaes that doesn't have a downstream version (like this one).   I'm happy to set these to something like 6.9  or 3.0 if its preferred

Comment (ekohl) [reply guides/common/attributes.adoc:9]: I'll merge it for now, but IMHO something we should address in master first.

Comment (melcorr) [reply guides/common/attributes.adoc:9]: Noooo these are for the upgrade guide :-( 
These were necessary

Comment (melcorr) [reply guides/common/attributes.adoc:9]: OK - good. From the email this wasn't clear. 
We need a target version to reference upgrades in upgrade guides.

Comment (jlsherrill) [reply guides/common/attributes.adoc:9]: @melcorr hey, to clarify do we need to change anything here in the 3.0 branch?    I wasn't suggesting we get rid of these values, i just didn't think they had any relevance to the  upstream docs for 3.0

Comment (ekohl) [reply guides/common/attributes.adoc:9]: @melcorr out of interest, why can't we use the ProjectVersion as the TargetVersion? And `2.4` doesn't make any sense, even if it doesn't end up being used, it's just confusing, right?

**Pull request merged**